StreamNode/WebClient/src/routes/dashboard/index.js

```diff
@@ -93,11 +95,13 @@ class Dashboard extends Component {
             return valid;
         }).map(audio => {
             //console.log(JSON.stringify(audio));
-            return (<Col span={this.state.isMobile ? 24 : 6} hidden={audio.hidden}>
+            return (<Col span={this.state.isMobile ? (this.state.isVertical ? 8 : 24) : (this.state.isVertical ? 2 : 6)} hidden={audio.hidden}>
```

Comment:
Try to use xs, sm, md, etc... for responsive resizment instead of react-responsive, for example:
<Col xs={(this.state.isVertical ? 8 : 24)} sm={this.state.isVertical ? 4 : 8} md={this.state.isVertical ? 3 : 8} lg={this.state.isVertical ? 3 : 6} xl={this.state.isVertical ? 2 : 6} xl={this.state.isVertical ? 2 : 6} xxl={this.state.isVertical ? 2 : 4} hidden={audio.hidden}>

Here is some example:
https://www.codegrepper.com/code-examples/javascript/responsive+grid+using+antd

And Here you can find documentation:
https://ant.design/components/grid/